sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/GenerateMigrationOutputTask.kt

```diff
@@ -72,7 +72,8 @@ abstract class GenerateMigrationOutputTask : SourceTask() {
           sourceFolders = parameters.sourceFolders.get(),
           dependencyFolders = emptyList(),
           moduleName = parameters.moduleName.get(),
-          properties = properties
+          properties = properties,
+          verifyMigrations = false
```

Comment:
```suggestion
          verifyMigrations = false,
```

Reply:
ktlint fails trailing commas in sqldelight right now